src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -787,6 +787,8 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="showOptions" value="Meta+," if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
          <shortcut refid="projectOptions" value="Shift+Meta+," if="org.rstudio.core.client.BrowseCap.isMacintosh()"/> 
          <shortcut refid="goToHelp" value="Ctrl+C Ctrl+V" disableModes="default,vim,sublime"/>
+         <shortcut refid="pasteAndIndent" value="Ctrl+Shift+V" if="!org.rstudio.core.client.BrowseCap.isMacintoshDesktop()"/>
+         <shortcut refid="pasteAndIndent" value="Meta+Shift+V" if="org.rstudio.core.client.BrowseCap.isMacintoshDesktop()"/>
```

Comment:
The intention here was to only ever register these commands in Desktop mode, as the server will 'just do the right thing' without these bindings. (We need to register them explicitly for desktop as otherwise the key combination is ignored)

Perhaps I should make this more clear by adding the notion of a 'desktop' vs. 'server' scoped command binding (as a new XML attribute). Note that combining conditions with `&&` doesn't work as the XML parser complains :-/